cypress/integration/layout-spec.js

```diff
@@ -0,0 +1,48 @@
+import { Preview } from '../page-objects/preview-page'
+import { goOffline, goOnline } from '../page-objects/helpers/network'
+
+const preview = new Preview()
+
+describe( 'Check the layout', () => {
+
+	beforeEach( 'Open the Test Page', () => {
+		cy.navigateToTestPage()
+	} )
+
+	it( 'Check the Wikipedia Preview in Standard with Image Layout', () => {
+		preview.getPreviewSpan().eq( 0 ).click()
+
+		preview.checkPreviewStandard()
+
+		preview.getHeaderClosebtn().click()
+	} )
+
+	it.skip( 'Check the Wikipedia Preview in Offline Layout', () => {
+		goOffline()
+
+		preview.getPreviewSpan().eq( 0 ).click()
+
+		preview.checkPreviewOffline()
+
+		preview.getHeaderClosebtn().click()
+
+		goOnline()
```

Comment:
If going back online is important for the other tests in this suite, doing something like `ensureOnline()` in the `beforeEach()` would be more reliable since exceptions in this test would skip the call to `goOnline()` in the last line.

Reply:
fixed